core/tests/protractor_utils/editor.js

```diff
@@ -430,25 +497,27 @@ var _selectRule = function(ruleElement, interactionId, ruleName) {
       optionElements[0].click();
     });
   });
+};
+
+// Before using this function ensure that expectValueToBe
+// (as declared in TODO below) is implemented in your parameter type editor.
+var expectRuleParametersToBe = function(interactionId, ruleName) {
+  var parameterValues = [];
+  for (var i = 2; i < arguments.length; i++) {
+    parameterValues.push(arguments[i]);
+  }
+
+  parameterTypes = _getRuleParameterTypes(interactionId, ruleName);
+  expect(parameterValues.length).toEqual(parameterTypes.length);
 
   // Now we enter the parameters
   for (var i = 0; i < parameterValues.length; i++) {
-    var parameterElement = ruleElement.all(
+    var parameterElement = element.all(
       by.css('.protractor-test-answer-description-fragment'
     )).get(i * 2 + 1);
     var parameterEditor = forms.getEditor(parameterTypes[i])(parameterElement);
-
-    if (interactionId === 'MultipleChoiceInput') {
-      // This is a special case as it uses a dropdown to set a NonnegativeInt
-      parameterElement.element(
-        by.tagName('button')
-      ).click();
-      parameterElement.element(
-        by.cssContainingText('.oppia-html-select-option', parameterValues[i])
-      ).click();
-    } else {
-      parameterEditor.setValue(parameterValues[i]);
-    }
+    // TODO: implement expectValueToBe in all parameterEditors.
```

Comment:
TODO(maitbayev)

(this just means that, if someone doesn't understand what this TODO means, they can ask you; it doesn't imply you have to write all of them).